requirements.txt

```
@@ -25,9 +25,10 @@ tenacity==8.0.1
 testimony==2.2.0
 unittest2==1.1.0
 wait-for==1.2.0
-wrapanapi==3.5.11
+
 
 # Get airgun, nailgun and upgrade from master
+git+https://github.com/omkarkhatavkar/wrapanapi.git@unpin_boto#egg=wrapanapi
```

can you update this one?
```suggestion
git+https://github.com/RedHatQE/wrapanapi.git@unpin_boto#egg=wrapanapi
```

---

As https://github.com/SatelliteQE/robottelo/pull/9370#discussion_r812747550 we will pin the latest version once the wrapanapi PR get merged

---

https://pypi.org/project/wrapanapi/   3.5.12 is available now

---

Done ! bump to 3.5.12